examples/ios/swift/Migration/README.md

```diff
@@ -0,0 +1,13 @@
+# Migration Examples
+
+The Migration project shows several examples of migrations and migration blocks.
+
+The purpose of this example is to provide a more in depth view of certain problems and pitfalls users might face when 
+migrations get more complex and the number of versions increases.
+
+## How to use the example
+
+You can build and run the project since this is the default state (the `Migration.xcconfig` is only used to create new examples).
```

Comment:
This isn't really true? The `Migration.xcconfig` file is so that the user has the option to step through the migration in each "phase" of their schema.

Reply:
I thought the newest version would be all a user would use since it migrated all prior version to the most recent version anyway. 0->5, 1->5, etc. (https://github.com/realm/realm-cocoa/blob/df/extend-migration-examples/examples/ios/swift/Migration/AppDelegate.swift#L53)
We do have multiple phases of `migrationBlock` though. Would a user want to step through earlier versions or just use that as a reference to understand how the 5 migration steps in v5 are done?

Reply:
They may want to see 0->1, 0->2, 2->3, 3->4, or any combination. The idea is to cover multiple use cases, for users of the SDK that want to understand basic migrations to complex migrations to partial migrations. Probably not something every user will use, but saying that the config file is only used to create new examples is incorrect